### ISSUE TITLE
How can I change UI timezone ? 

### ISSUE THREAD

**Issue by iseehk** [gessnerfl/fake-smtp-server#265]
Hi When I check the console log , it shows current local time,
when I open the browser and check the "sent time", it is UTC time zone (I guess, not local time) . I tried following but in vain:

```
%JAVA_HOME17%\bin\java -Duser.timezone="Asia/Hong_Kong" -jar fake-smtp-server-2.1.2.jar  -Dspring.config.location=foo\application.properties  
```

`application.properties`

1.  add "spring.jpa.properties.hibernate.jdbc.time_zone=Asia/Hong_Kong" in application.properties
2.  add spring.datasource.url=jdbc:h2:mem:mail;TIME ZONE=Asia/Hong_Kong

Though it is a FAKE Smtp server . but it is good to show the current timezone. 
any advice ?

Thank you.

**Comment by gessnerfl** [comment]
@iseehk this is currently not supported. Planned for a future release

**Comment by gessnerfl** [comment]
@iseehk there was an issue with the handling of timezones in the backend. I changed the backend to work with timestamps including timezone information using UTC timezone. The APIs are also changed to return UTC based timestamps compliant with [ISO 8601](https://de.wikipedia.org/wiki/ISO_8601). With this the timestamps are properly handled in the user interface and rendered in the timezone of the browser.

**Comment by gessnerfl** [comment]
The issue with local timezone will be fixed with version 1.5.2. Please reopen if the issue persists or the fix does not meet your expectation.
